apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -246,21 +254,27 @@ export const getAzureConfigFromEnvironment: RE.ReaderEither<
       appInsights,
       cosmos,
       entityConfigurationStorageContainerName,
+      pidIssuerRevokeApiQueueName,
       revocationQueueName,
       storageAccountConnectionString,
     }) => ({
       appInsights,
       cosmos,
-      queue: {
-        walletInstanceRevocation: {
-          connectionString: storageAccountConnectionString,
-          name: revocationQueueName,
-        },
-      },
       storage: {
         entityConfiguration: {
           containerName: entityConfigurationStorageContainerName,
         },
+        walletInstances: {
+          connectionString: storageAccountConnectionString,
+          queues: {
+            pidIssuerRevokeApi: {
+              name: pidIssuerRevokeApiQueueName,
+            },
+            walletInstanceRevocation: {
+              name: revocationQueueName,
```

Comment:
I assume you are already inside the Wallet Instances object
```suggestion
            revocation: {
              name: revocationQueueName,
```